ls8/cpu.c

```diff
@@ -4,21 +4,27 @@
 
 #define DATA_LEN 6
 
-/** --> Northbridge goes here <--
- * TODO: We'll make use of these helper functions later.
+/**
+ * --> Northbridge goes here <--
  */
-// cpu_ram_read()
-int *cpu_ram_read(struct cpu *cpu)
+int cpu_ram_read(struct cpu *cpu, int address)
```

Comment:
by far the most sophisticated verison of `cpu_ram_read()` I've seen so far. 3 stars